app/scripts/modules/core/pipeline/config/stages/executionWindows/ExecutionWindowDayPickerComponent.js

```diff
@@ -0,0 +1,89 @@
+import angular from 'angular';
```

Comment:
Should be renamed `executionWindowDayPicker.component.js` to be consistent with other components.